### DISCUSSION TITLE
is there a reason the volume overlay is below the settings overlay

### DISCUSSION THREAD

**Discussion by daifkyou** [ppy/osu#21025] [Q&A] [answered]
seems kinda weird but i dont want to open an issue bc im dumb and it seems pretty obvious

i always wondered what magic search engine people use to find issues other than google and gh
maybe its all in their heads?

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/11077.

**Reply by daifkyou** [reply]
k sorry
